Whereis doc

aswarner opened the pull request:
# Advanced HW 5 submission

Closes #12

Name: Anastasia Warner
Uniqname: aswarner

## Description of Changes
Added basic documentation for whereis, along with examples of how to use options and flags. Also includes where the command searches

stevietriesenberg commented:
Hi Anastasia,
Your contribution should only change one file, but it looks like you changed 808 files.  Your documentation file looks like it has some merge conflict markers left in as well as some markdown formatting issues.  If you could fix these three things, we will award you credit!

aswarner commented:
Hi Stevie,
I made some changes and I want to make sure I'm doing the right thing
before creating another pull request.

First, I fixed the 808 files changed problem. That should be gone. I
started clean by pulling again from the main branch, which now has some
merged in changes from other students' pull requests. I used this as my
base. That should be okay, right?

I changed my markdown syntax by adding empty lines between items and
changing a random ellipsis to an actual markdown-syntax divider. However, I
don't have a way to verify my additions are correct because when I try to
follow the instructions in the README to run "python setup.py develop", but
my terminal says :
 "can't create or remove files in install directory ...
    [Errno 13] Permission denied:
'/usr/local/lib/python2.7/dist-packages/test-easy-install-14945.write-test'
The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:
    /usr/local/lib/python2.7/dist-packages/ ...
Perhaps your account does not have write access to this directory? ... "
I don't know how to resolve this so I cannot start up the simulated website
with my changes. Do you have any advice?

Thanks,
Anastasia



On Fri, Mar 22, 2019 at 10:06 AM Stevie Triesenberg <
notifications@github.com> wrote:

> Hi Anastasia,
> Your contribution should only change one file, but it looks like you
> changed 808 files. Your documentation file looks like it has some merge
> conflict markers left in as well as some markdown formatting issues. If you
> could fix these three things, we will award you credit!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/csprag/reference/pull/91#issuecomment-475634021>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ApteTQhJbi95p_JeYgJhmwCMYlZlD6SAks5vZON-gaJpZM4a__RD>
> .
>
